### PR TITLE
Add execution duration

### PR DESCRIPTION
**Pull request by manucamejo**

### CODE REVIEW

**Comment by mFragaBA** [comment]
is there a PR in the [curse_of_myrra repo](https://github.com/lambdaclass/curse_of_myrra) updating the `messages.proto` file and the csharp related files?

**Comment by manucamejo** [comment]
> is there a PR in the [curse_of_myrra repo](https://github.com/lambdaclass/curse_of_myrra) updating the `messages.proto` file and the csharp related files?

Yes, [here](https://github.com/lambdaclass/curse_of_myrra/pull/1191)

**Review by ncontinanza — DISMISSED** [review]
LGTM!

**Review by mFragaBA — CHANGES_REQUESTED** [review]

File: messages.proto
[the remark this answers is not in the document]
Comment (mFragaBA) [reply messages.proto:1]: we talked about this and will discuss with @AminArria but it shouldn't block the merge of this PR. We'll iterate on a later PR

**Review by mFragaBA — APPROVED** [review]
LGTM!

**Pull request merged**